packages/augur-ui/src/assets/styles/_animations.less

```diff
@@ -32,8 +32,8 @@ body {
   .animation(flash-slow @animation-speed-slow @animation-default-type infinite alternate);
 
   .keyframes(flash-slow, {
-    0% { background: fade(@color, @amount-subtle) }
-    100% { background: fade(@color, @amount-light) }
+    0% { background: fade(@color, 2%) }
```

Comment:
@JohnDanz Why don't you modify `@amount-subtle` and `@amount-light` directly?
If it's being used elsewhere, I suggest creating another variable, specially if the same values (2% and 15%) will be used throughout the code.

Reply:
looks like @amount-subtle isn't needed at all and isn't used anywhere else.